### PR TITLE
AreaMomentOfInertia operator / overload

### PR DESCRIPTION
**Pull request by lwestfall**

### CODE REVIEW

**Comment by lwestfall** [comment]
I'm unsure what I did wrong for the first 4 commits to be included in this PR. I only intended to include the final one "af39705". If I need to do something and resubmit please let me know.

Thanks

**Comment by angularsen** [comment]
Hey and thanks for contributing. My best guess what happened is that your branch `areamomentofinertia-operator-overloads` was based on your own `master` branch, not `angularsen` repo's `master` and you had those extra commits from there.

This is how I recommend you fix it with git from console, details may vary depending on how you have it set up locally and if you prefer using git gui tools:
1. Add `angularsen` repo as a remote: `git remote add angularsen https://github.com/angularsen/UnitsNet`
2. `git fetch` to download everything
3. `git checkout areamomentofinertia-operator-overloads` to checkout your branch locally
4. `git rebase -i angularsen/master` to interactively rebase your branch on top of angularsen's master
5. Delete all lines except the commit you want to keep, it should say `pick <commit> <message>`, save and exit (type `:x` if you are using vim editor)
6. `git log` to verify that you now only have this commit in addition to `angularsen/master`'s commits
7. `git push --force` to overwrite your remote branch and effectively update the commits of this PR

Git can be confusing, but I'm happy to help and here are some docs too:
https://help.github.com/en/articles/creating-a-pull-request-from-a-fork
https://help.github.com/en/articles/working-with-forks

File: UnitsNet/CustomCode/Quantities/AreaMomentOfInertia.extra.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet
+{
+    public partial struct AreaMomentOfInertia
+    {
+        /// <summary>Get <see cref="Volume"/> from <see cref="AreaMomentOfInertia"/> divided by <see cref="Length"/>.</summary>
+        public static Volume operator /(AreaMomentOfInertia areaMomentOfInertia, Length length)
```
Comment (angularsen) [inline UnitsNet/CustomCode/Quantities/AreaMomentOfInertia.extra.cs]: Can you add a test case for this in 
https://github.com/angularsen/UnitsNet/blob/master/UnitsNet.Tests/CustomCode/AreaMomentOfInertiaTests.cs#L28-L41 ?

See other classes in the same CustomCode folder for examples, such as this one:
https://github.com/angularsen/UnitsNet/blob/master/UnitsNet.Tests/CustomCode/LengthTests.cs#L111-L116

Comment (lwestfall) [reply UnitsNet/CustomCode/Quantities/AreaMomentOfInertia.extra.cs]: Test case and commit added! 😎

**Comment by angularsen** [comment]
Awesome!
Nuget is on the way out.

[Release UnitsNet/4.39.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.39.0)

**Pull request merged**